python_src/src/lamp_py/startup/start_ingestion.py

```diff
@@ -119,15 +88,20 @@ def main() -> None:
         schedule.run_pending()
         time.sleep(1)
 
-    # send shutdown signal to rds process and wait for finish
-    # not sure if we'll ever get here the way we're using @schedule
-    # metadata_queue.put(None)
-    # _rds_process.join()
-
 
-if __name__ == "__main__":
-    logging.info("Starting Ingestion Container")
+def start() -> None:
+    """configure and start the ingestion process"""
+    # setup handling shutdown commands
     signal.signal(signal.SIGTERM, handle_ecs_sigterm)
+
+    # configure the environment
     load_environment()
+    os.environ["SERVICE_NAME"] = "ingestion"
     validate_environment()
+
+    # run the main method
     main()
+
```

Comment:
created a `start`function that is called in the poetry script. its also called when we run this as a script.